repositories.yaml

```diff
@@ -167,6 +167,22 @@ applications:
       - v1_name: firefox-desktop
         app_id: firefox-desktop
 
+  - app_name: firefox_desktop_background_updater
+    canonical_app_name: Firefox for Desktop Background Updater
+    app_description: >-
+      The desktop version of Firefox's background update tasks. This is
+      considered a seperate application because it has its own client
+      id and sends its own version of the standard pings.
+    metrics_files:
+      - toolkit/mozapps/update/update_metrics.yaml
+    ping_files:
+      - toolkit/mozapps/update/update_pings.yaml
+    dependencies:
+      - glean-core
```

Comment:
The [original bug suggests](https://bugzilla.mozilla.org/show_bug.cgi?id=1704878#c0) we should specify service-glean as a dependency, but that doesn't make a lot of sense to me for a new application. That functionality is android-specific and we only include it for desktop firefox for backwards compatibility reasons.

Reply:
That sounds good to me.